src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -5257,6 +5267,274 @@ def remove_github_actions(cmd, resource_group, name, repo, token=None, slot=None
     return "Disconnected successfully."
 
 
+def add_functionapp_github_actions(cmd, resource_group, name, repo, runtime=None, runtime_version=None, token=None,  # pylint: disable=too-many-statements,too-many-branches
+                                   slot=None, branch='master', build_path=".", login_with_github=False, force=False):
+    if not token and not login_with_github:
+        raise_missing_token_suggestion()
+    elif not token:
+        scopes = ["admin:repo_hook", "repo", "workflow"]
+        token = get_github_access_token(cmd, scopes)
+    elif token and login_with_github:
+        logger.warning("Both token and --login-with-github flag are provided. Will use provided token")
+
+    # Verify resource group, app
+    site_availability = get_site_availability(cmd, name)
+    if site_availability.name_available or (not site_availability.name_available and
+                                            site_availability.reason == 'Invalid'):
+        raise ResourceNotFoundError(
+            "The Resource 'Microsoft.Web/sites/%s' under resource group '%s' "
+            "was not found." % (name, resource_group))
+    app_details = get_app_details(cmd, name)
+    if app_details is None:
+        raise ResourceNotFoundError(
+            "Unable to retrieve details of the existing app %s. Please check that the app is a part of "
+            "the current subscription" % name)
+    current_rg = app_details.resource_group
+    if resource_group is not None and (resource_group.lower() != current_rg.lower()):
+        raise ResourceNotFoundError("The webapp %s exists in ResourceGroup %s and does not match the "
```

Comment:
same comments as above I would test this for ASP & App on different RGs.

Reply:
Just confirmed that this still works when the ASP and app are in different groups